tools/sst_dump_tool.cc

```diff
@@ -588,6 +593,17 @@ int SSTDumpTool::Run(int argc, char** argv) {
       }
     }
 
+    if (command == "verify") {
```

Comment:
what's the difference between `--command=verify` and `--command=check` with `--verify_checksum`? can we just enable `--verify_checksum` by default (it always confused me why we don't)?